rest-api/tools/csv_importer.py

```diff
@@ -58,15 +60,21 @@ def run(self, filename, dry_run):
             changed = self._update_entity(entity, existing_entity, session, dry_run)
             if changed:
               updated_count += 1
+            elif self.skip is True:
+              skip_count += 1
+              self.skip = False
             else:
               matched_count += 1
           else:
             self._insert_entity(entity, existing_map, session, dry_run)
```

Comment:
Need to handle skip case here, too.

Reply:
regarding above, the skip is only to add to the skipped_count which is called on _entity_from_row (prior to any geocoding). So that the count will be accurate during the final logging.info